hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/AutoCreationLogic.java

```diff
@@ -196,34 +197,52 @@ public Pair<ResponseCodeEnum, Long> create(
                     "Cannot auto-create an account from unaliased change " + change);
         }
 
-        // checks tokenAliasMap if the change consists an alias that is already used in previous
-        // iteration of the token transfer list. This map is used to count number of
-        // maxAutoAssociations needed on auto created account
-        analyzeTokenTransferCreations(changes);
+        TransactionBody.Builder syntheticCreation;
+        HederaAccountCustomizer customizer;
+        if (alias.size() == EntityIdUtils.EVM_ADDRESS_SIZE) {
```

Comment:
Q: where's the feature flag for this?
Or is it gated already or in another PR at a higher level?

Reply:
Looks like this will be addressed here https://github.com/hashgraph/hedera-services/pull/4044

Reply:
Yes, the feature flags will be added in https://github.com/hashgraph/hedera-services/pull/4044, I will be updating the code here to use the needed one.